nextflow_schema.json

```diff
@@ -1,4 +1,4 @@
-// TODO: Update the description
+
```

Comment:
You can remove this line, but it's worth adding the input CSV here as an option so it populates the UI on Tower. See the phylorthology `nextflow_schema.json`

I should update this comment, as this is not only about the description but also for the Nextflow Tower/nf-core UI